dts/bindings/ethernet/atmel,sam-gmac.yaml

```diff
@@ -41,6 +41,11 @@ properties:
           - "mii"
         default: "rmii"
 
+    peripheral-id:
+      type: int
+      description: Peripheral ID
+      required: false
```

Comment:
I believe this should be a required property, i.e. we need `true` here. Without the correct periph id value the driver will not work.

Reply:
That would make any SAM0 devices (e.g. SAM E5x) using this to specify dummy `peripheral-id` here.